dev/com.ibm.ws.repository.resolver/src/com/ibm/ws/repository/resolver/RepositoryResolver.java

```diff
@@ -370,14 +390,78 @@ public Collection<List<RepositoryResource>> resolve(String toResolve) throws Rep
      * @throws RepositoryResolutionException If the resource cannot be resolved
      */
     public Collection<List<RepositoryResource>> resolveAsSet(Collection<String> toResolve) throws RepositoryResolutionException {
-        return resolve(toResolve, ResolutionMode.DETECT_CONFLICTS);
+        return resolve(toResolve, null, ResolutionMode.DETECT_CONFLICTS);
+    }
```

Comment:
Should this version of the method be deprecated?

Reply:
yes I added to javadoc